lib/matplotlib/artist.py

```diff
@@ -898,6 +900,89 @@ def set_zorder(self, level):
         self.pchanged()
         self.stale = True
 
+    def get_top_margin(self):
```

Comment:
Why be inconsistent with everything else?  I know traits will eventually get us there, but we're not there yet...